VersionMiner/Exceptions/NoXMLElementException.cs

```diff
@@ -0,0 +1,40 @@
+﻿// <copyright file="NoXMLElementException.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace VersionMiner.Exceptions;
+
+/// <summary>
+/// Thrown if a XML element does not contain a value.
```

Comment:
Use different article
```suggestion
/// Thrown if an XML element does not contain a value.
```

Reply:
I did change it but I was wondering, why is it `an` instead of `a`?
The next "word" `XML` does not start with a vowel.